modules/combined/examples/phase_field-mechanics/grain_texture.i

```diff
@@ -0,0 +1,165 @@
+# This simulation predicts GB migration of 8 grains and outputs grain texture information
+# Mesh adaptivity is not used so that the VectorPostprocessor's output will be uniform
+# Time step adaptivity is used
+# An AuxVariable is used to calculate the grain boundary locations
+# Postprocessors are used to record time step
+
+[Mesh]
+  # Mesh block.  Meshes can be read in or automatically generated
+  type = GeneratedMesh
+  dim = 2 # Problem dimension
+  nx = 100 # Number of elements in the x-direction
+  ny = 100 # Number of elements in the y-direction
```

Comment:
The input file you are looking at isn't part of any test. It is just an
example. The gold files are only 101 and 201 lines long.

On Wed, May 18, 2016 at 8:57 AM, Cody Permann <notifications@github.com>
wrote:

> In modules/combined/examples/phase_field-mechanics/grain_texture.i
> <https://github.com/idaholab/moose/pull/7009#discussion_r63719029>:
>
> > @@ -0,0 +1,165 @@
> > +# This simulation predicts GB migration of 8 grains and outputs grain texture information
> > +# Mesh adaptivity is not used so that the VectorPostprocessor's output will be uniform
> > +# Time step adaptivity is used
> > +# An AuxVariable is used to calculate the grain boundary locations
> > +# Postprocessors are used to record time step
> > +
> > +[Mesh]
> > +  # Mesh block.  Meshes can be read in or automatically generated
> > +  type = GeneratedMesh
> > +  dim = 2 # Problem dimension
> > +  nx = 100 # Number of elements in the x-direction
> > +  ny = 100 # Number of elements in the y-direction
>
> Good point - 100x100 is pretty large for a test. It might seem fast on
> your local workstation but it'll be dog slow in debug mode on a linux box
> with all the assertion tests. I'd try to cut it down as much as you can.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/7009/files/e86200f4217fc0dca3ed746e58ae76897ca055e1#r63719029>
>